### PR TITLE
Add mandarin translations

### PR DESCRIPTION
**Pull request by amaxama**
### What
Added Mandarin Translations

### Why
Issue #246 

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [x] Tapping on a marker on the map displays information about the marker in a popup.
- [x] Tapping the "Show list of locations" button replaces the map view with a list view.
- [x] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [x] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [x] Changing the language to spanish changes things on the page.
- [x] Clicking the Help/Info button opens and closes a menu with information.
- [x] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [x] Safari

### CODE REVIEW

**Comment by amaxama** [comment]
Waiting on 3 quick things from translators

File: src/i18n/man.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  "language": "语言",
+  "welcome": "欢迎",
+  "welcome_blurb": "双城相助 （Twin Cities Mutual Aid）是关于明尼阿波利斯和圣保咯两座城人民市相助地点的最新目录。",
+  "lang_select": "选择语言",
+  "page_title": "双城相助地图",
+  "show_list_button": "显示地点单子",
+  "hide_list_button": "关闭地点单子",
+  "sort_by": "用类目分别",
+  "sort_by_urgent_need": "紧急请求",
+  "sort_by_name": "",
```
Comment (mc-funk) [inline src/i18n/man.json:14]: are we just missing this translation?

Comment (mc-funk) [reply src/i18n/man.json:14]: Looks addressed in comment, missed that. There seemed to be 4, though, including "time_future"

File: docs/LANGUAGE_TRANSLATION.md [hunk 2]
```diff
@@ -2,19 +2,25 @@
...
+      - [Create a new translation file](#create-a-new-translation-file)
+      - [Adding a new flag image](#adding-a-new-flag-image)
+      - [Moment.js](#momentjs)
+      - [Enabling the language](#enabling-the-language)
+      - [Add language checklist](#add-language-checklist)
+  - [Technical Details](#technical-details)
+    - [Adding a term in markup (`data-translation-id`)](#adding-a-term-in-markup-data-translation-id)
+    - [Getting a translated term in js code (`get`)](#getting-a-translated-term-in-js-code-get)
+    - [Setting the current language (`language`)](#setting-the-current-language-language)
+    - [Running translation substitution (`translate`)](#running-translation-substitution-translate)
```
Comment (amaxama) [inline docs/LANGUAGE_TRANSLATION.md:23]: Hmmm, seems this update happened automatically?

File: src/i18n/man.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  "for_receiving": "收到",
+  "seeking_volunteers_badge": "寻找志愿者",
+  "seeking_money": "寻找捐款",
+  "seeking_money_link": "来捐款！",
+  "urgent_need": "急用",
+  "notes": "笔记",
+  "last_updated": "最近更新",
+  "aid_distribution_hours": "相助分送时间",
+  "aid_receiving_hours": "相助接受时间",
+  "time_future": "",
```
Comment (amaxama) [inline src/i18n/man.json:61]: This does not appear to have a translation, but I also don't see anywhere were it's used anyway...

File: src/js/translator.js [hunk 4]
```diff
@@ -4,32 +4,34 @@
  * Using local translation files located under /src/i18n/*.js
  */
 
-import eng from './../i18n/eng.json'
```
Comment (amaxama) [inline src/js/translator.js:7]: Just alphabetized this

Comment (mc-funk) [reply src/js/translator.js:7]: Saw that, loooooove it

File: docs/LANGUAGE_TRANSLATION.md (hunk 2 above)
Comment (amaxama) [reply docs/LANGUAGE_TRANSLATION.md:23]: @mc-funk I think it's being an actual table of contents... in my IDE it says `Table of Contents (up to date)` like this, and on master it says `Table of Contents (up to date)` - I think I have a formatter in my IDE that probably made it up to date so should be okay!

**Review by comeoneileen — APPROVED** [review]
Nice! Thanks so much for tackling this.

**Pull request merged**